### DISCUSSION TITLE
Ejecutable lossless-cut-master

### DISCUSSION THREAD

**Discussion by RubberSound** [mifi/lossless-cut#2254] [Support]
Hola. Descargué el archivo  lossless-cut-master.zip, lo descomprimí, y en todos los archivos que figuran, no hay ninguno ejecutable.
Entonces: ¿Cómo se ejecuta el programa? Gracias. (Adjunto imágen con todos los archivos que contiene la carpeta descargada).
![lossless-cut-master](https://github.com/user-attachments/assets/3296fbdb-5987-4436-b2fe-fb74691693b6)

**Comment by mifi** [comment]
See https://github.com/mifi/lossless-cut?tab=readme-ov-file#download
